libraries/default.rb

```diff
@@ -38,5 +38,9 @@ def format_kernel_parameters  # rubocop:disable all
 
       rendered.each { |r| r.prepend('    ') }.join(",\n")
     end
+
+    def format_ssl_versions
+      Array(node['rabbitmq']['ssl_versions']).map { |n| "'#{n}'" }.join(',')
```

Comment:
I find this less than obvious. How about we simply set default to an empty array instead of `nil`? `nil` seems illogical to me.